nautobot/core/api/metadata.py

```diff
@@ -246,8 +263,16 @@ def determine_view_options(self, request, serializer):
             column_data = processor._get_column_properties(field, field_name)
             fields.append(column_data)
 
+        # Construct the advanced tab table fields.
+        advanced_data = [
+            {
+                "Object Details": {"fields": advanced_fields},
+            }
+        ]
+
         return {
             "retrieve": self.determine_detail_view_schema(serializer),
+            "advanced": advanced_data,
             "list_display_fields": list_display,
             "fields": fields,
```

Comment:
Can we think about how to more generally structure this data for views that might have more tabs than just the default and Advanced tabs?